Gemfile

```diff
@@ -6,8 +6,8 @@ gem 'asciidoctor', '1.5.6.2'
 gem 'json'
 gem 'awesome_print'
 
-gem 'asciidoctor-epub3', '~> 1.5.0.alpha.9'
-gem 'asciidoctor-pdf', '~> 1.5.0.beta.8'
+gem 'asciidoctor-epub3', '1.5.0.alpha.10'
```

Comment:
Please, try alpha-13, it has a bunch of bugs fixed compated to alpha-10 and no known regressions.

Reply:
OK. Will force-push a corrected version.